addon/models/learner-group.js

```diff
@@ -144,7 +145,7 @@ export default Model.extend({
   allDescendants: computed('children.@each.allDescendants', async function(){
     const descendants = [];
     const children = await this.get('children');
-    descendants.pushObjects(children.toArray());
```

Comment:
i don't understand this change. `this.get('children')` should a return a PromiseArray, so calling `toArray()` on it seems necessary here.

Reply:
Thanks, I got a little bit overzealous with my first replacement. I thought I had put them all back, but I missed this one.